aip/client-libraries/4235.md

```diff
@@ -7,17 +7,17 @@ created: 2024-03-07
 # Automatically populate fields in the request message
 
 For APIs that leverage request idempotency as described via [AIP-155], APIs may
-choose to have the client libraries automatically populate the `request_id`
-if it is not already set by the customer.
+choose to have the client libraries automatically populate fields such as 
+`request_id` if they are not already set by the customer.
```

Comment:
If `"they are not already set by the customer"` is a requirement, should we also include it in the list of conditions in the guidance section?

```
For a field to be automatically populated, **all** the conditions below
**must** be true:

- The field **must** not already be set
...